scripts/travis.sh

```diff
@@ -5,19 +5,14 @@ source "${0%/*}/helpers.sh"
 
 export CHROME_HEADLESS=true
 
-# Run the yarn install to pull in test dependencies
-fold yarn_install yarn install
-
-# Lint the markdown
-# fold yarn_markdown_check yarn markdown-lint
-
 # Check the links
-fold yarn_check_links yarn check-links
+# Note: yarn build should already have been run in before_script travis step
+# so we don't have to run it again here
+fold yarn_check_links yarn broken-link-checker:internal
```

Comment:
We build in the `.travis.yml` `build` step now, so we only need to run the actual link checking part (previous `yarn check-links` also does a build)